### DISCUSSION TITLE
Is there a discord server or somewhere else there is a community for discussion?

### DISCUSSION THREAD

**Discussion by nlighteneddesign** [firefly-iii/firefly-iii#5314] [General]
I'm curious where to share configurations and tips.

**Comment by JC5** [comment]
Check out the readme file, the link to Gitter is right there. 👍

**Reply by nlighteneddesign** [reply]
Ah I always forget about gitter, I ctrl+f for discord. Thanks!

**Comment by NDCallahan** [comment]
I'd love to see a discord. it's so hard to keep up with all of these individual community types, lol (first world problem, I know).

**Reply by JC5** [reply]
Sorry, I'll never join Discord with Firefly III.

**Comment by nlighteneddesign** [comment]
I think it is safe to close this issue since there is an answer here and the gitter is in the readme.
